Directory.build.props

```diff
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup>
     <Copyright>Copyright (c) .NET Foundation and Contributors</Copyright>
-    <PackageLicenseUrl>https://opensource.org/licenses/ms-pl</PackageLicenseUrl>
+    <PackageLicenseExpression>MS-PL</PackageLicenseExpression>

```

Comment:
I think in addition to using this new property we should add the license to the NuGet package too:

```msbuild
<ItemGroup>
    <None Include="licenses\LICENSE.txt" Pack="true" PackagePath=""/>
</ItemGroup>
```

Reply:
What would that help?

Reply:
You wouldn't have to go somewhere else to get the actual license for the library. Also for the license to actually cover the distributed code, it must be accompanied with it.